draft-ietf-dprive-dnsoquic.md

```diff
@@ -663,29 +702,61 @@ attacker might replay it at chosen times, several times.
+the attacker can choose the time at which the 0-RTT data will be replayed.
+We believe that the freshness tests recommended in {{connection-resume-and-0-rtt}}
+significantly reduce the time range in which 0-RTT data can be
+replayed, and thus significantly reduce the potential of such
+attacks. The maximum delay parameter stated in {{connection-resume-and-0-rtt}}
+is 30 seconds. We believe this is consistent with commonly used values
+of the cached records TTL, and thus sufficiently small to impede
+most replay attacks.
 
 ## Privacy Issues With Session Resume
 
 The QUIC session resume mechanism reduces the cost of re-establishing sessions
 and enables 0-RTT data. There is a linkability issue associated with session
-resume, if the same resume token is used several times, but this risk is
-mitigated by the mechanisms incorporated in QUIC and in TLS 1.3. With these
-mechanisms, clients and servers can cooperate to avoid linkability by third
-parties. However, the server will always be able to link the resumed session to
-the initial session. This creates a virtual long duration session. The series
-of queries in that session can be used by the server to identify the client.
-
-Enabling the server to link client sessions through session resume is probably
-not a large additional risk if the client's connectivity did not change between
-the sessions, since the two sessions can probably be correlated by comparing
-the IP addresses. On the other hand, if the addresses did change, the client
-SHOULD consider whether the linkability risk exceeds the performance benefits.
-This evaluation will obviously depend on the level of trust between client and
-server.
+resume, if the same resume token is used several times. Attackers on path
+between client and server could observe repeated usage of the token and
+use that to track the client over time or over multiple locations. 
+
+The session resume mechanism allows servers to correlate the resumed sessions
+with the initial sessions, and thus to track the client. This creates a virtual
+long duration session. The series of queries in that session can be used by the
+server to identify the client. Servers can most probably do that already if
+the client address remains constant, but session resume tickets also enable
+tracking after changes of the client's address.
+
+The recommendations in {{connection-resume-and-0-rtt}} are designed to
+mitigate these risks. Using session resume tickets only once mitigates
+the risk of tracking by third parties. Refusing to resume session if addresses
+change mitigates the risk of tracking by the server.
+
+## Privacy Issues With New Tokens
+
+QUIC specifies address validation mechanisms in section 8 of {{!RFC9000}}.
+The NEW TOKEN mechanism allow QUIC servers to provide clients with
+address validation tokens, and avoid an extra RTT when address
+validation is required. The tokens are tied to an IP address. QUIC
```

Comment:
```suggestion
Use of an address validation token allows QUIC servers to avoid an extra RTT
for new connections. Address validation tokens are typically tied to an IP address. QUIC
```